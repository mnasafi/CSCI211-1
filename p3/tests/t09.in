insert
b
url-b
comment-b
2
2
insert
a
url-a
comment-a
1
1
insert
c
url-c
comment-c
3
3
insert
d
url-d
comment-c
4
4
print
