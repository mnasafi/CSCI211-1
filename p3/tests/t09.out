a, url-a, comment-a, 1, *
b, url-b, comment-b, 2, **
c, url-c, comment-c, 3, ***
d, url-d, comment-c, 4, ****
